tests/auth_mock.py

```diff
@@ -0,0 +1,331 @@
+import json
```

Comment:
Imports are out of order; this would be better (standard library, then third-party, then internal to the package):
```
import datetime
import json
import re
import urlparse

import httmock
import pytest

from sheepdog import test_settings
from sheepdog.auth import AuthDriver
```

Reply:
fixed